nuitka/tools/quality/auto_format/YamlFormatter.py

```diff
@@ -0,0 +1,234 @@
+from collections import OrderedDict
+from copy import copy
+from pathlib import Path
+
+import yaml
+
+MASTER_KEYS = [
```

Comment:
Would it be possible to get them from the schema file instead?

Reply:
Yes, I can do it.